### DISCUSSION TITLE
hass + mpd + ikea remote + beets

### DISCUSSION THREAD

**Discussion by trapd00r** [adnanh/webhook#633] [Show and tell]
This has to be one of the most empowering pieces of software I've stumbled upon.

I've (so far) used it to connect my homeassistant instance with a cheap ikea remote that controls my music way beyond the normal next/prev/play functionality one would expect.

Details here: https://github.com/trapd00r/hass-mpd-ikea

**Comment by adnanh** [comment]
Amazing stuff! :) This is exactly the kind of projects I had in mind when I was bootstrapping this tool. :)

If you want, you can create a PR to the webhook's README.md file with the link to your repository :)
